onnx_trt_backend.cpp

```diff
@@ -389,7 +404,7 @@ onnxStatus GraphRep::InitIO(uint32_t inputsCount,
         return ONNXIFI_STATUS_UNIDENTIFIED_NAME;
       }
       if (auto ret =
-              CheckAndBindTensor(dims, *it->second) != ONNXIFI_STATUS_SUCCESS) {
+              CheckAndBindTensor(dims, *it->second, true) != ONNXIFI_STATUS_SUCCESS) {
```

Comment:
Why? A comment would be good

Reply:
TRT shape is always NCHW, 4 dimension. While practically, the output of FC is 2d, as suggested by shape inference. We allow this to happen as long as the element sizes are the same. It's like an reshape.